tests/infer/mcmc/test_hmc.py

```diff
@@ -16,18 +15,6 @@
 logger = logging.getLogger(__name__)
 
 
-def mark_jit(*args, **kwargs):
-    jit_markers = kwargs.pop("marks", [])
-    jit_markers += [
-        pytest.mark.skipif('CI' in os.environ,
-                           reason='slow test'),
-        pytest.mark.skipif('CUDA_TEST' in os.environ,
```

Comment:
- Have we tested that these run (and finish within reasonable time) on CUDA?
 - I am also not sure if we should run all HMC JIT tests on CI. HMC tests even with the JIT flag are quite expensive. The additional signal by enabling the JIT flag is low (it covers the same code path essentially) and may not justify adding several minutes to unit test times. I think whitelisting one or two should be good enough for coverage, and we can change the reason above to something other than "slow test", if the slowness issues are fixed i.e. JIT is not slower than the non JIT test.

Reply:
Oppss... I have interpreted this marker wrongly. I thought they have no functionality now. >"<